### PR TITLE
Added Image and input

### PR DESCRIPTION
**Pull request by abutaher11**
checkout the links for the components--

https://pvggp.csb.app/Components/Image/Image.html

https://pvggp.csb.app/Components/Input/Input.html

### CODE REVIEW

File: Components/Input/Input.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@import url(/style.css);
+
+
+.input-group {
+    font-size: 1rem;
+    line-height: 20px;
+    margin: 0 20px 10px;
+    width: 70%;
+}
```
Comment (AnkurChunekar) [inline Components/Input/Input.css:9]: relative unit like rems can be used here

**Review by AnkurChunekar — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -18,3 +18,16 @@
     --off-white : #F6F9FC;
     --active-color : #1EA7FD;
 }
+
+.example-div{
+    border-radius: 4px;
+    /* background-color: #F5F7F9; */
```
Comment (Ayushi20-19) [inline style.css:24]: u can remove unwanted commented code.

**Review by Ayushi20-19 — APPROVED** [review]

File: Components/Input/Input.css (hunk 1 above)
Comment (abutaher11) [reply Components/Input/Input.css:9]: I'll keep that in mind

File: style.css (hunk 2 above)
Comment (abutaher11) [reply style.css:24]: thanks for noticing. I'll be using them in future that's why commented instead of removing

**Pull request merged**